set_bounds.c

```diff
@@ -6,25 +6,20 @@
 /***
  * The point of this program is to demonstrate the failing of a program
  * that tries to extend the bounderies of a capable pointer.
+ * Compared to the simpler 'bounds.c', here we try to modify the bounds
+ * explicitly.
  ***/
+
 int main()
 {
 	int32_t array[16] = {0};
-
 	uint32_t bounds = 64;
-	puts("bounds(64): ");
+
+	puts("Bounds [Choose a value greater than 64]:");
 	if (0 == scanf("%u", &bounds))
 	{
 		error("Extraneous input");
 	}
-
+	// trying to extend the bounds generate an exception
```

Comment:
Sorry for the nitpicking: we capitalise comments (so "Trying"); and put a full stop at the end. And I think "generate" -> "generates". If you make those changes in a new commit and push them, the best trick we've found is to then add a comment after my comment saying "Fixed in XYZ" where "XYZ" is the commit hash.

Reply:
Fixed in 0ce3570